.github/workflows/ex_doc_version_check.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: ex_doc_version_check
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+
```

Comment:
```suggestion
  workflow_dispatch:
  
```

Reply:
This is to let maintainers run the workflow, from e.g. `main` by pressing a button and choosing a branch.